README.rst

```diff
@@ -13,14 +13,14 @@ Examples:
 Pulling raw data from Clarity (still needs formatting work):
 
 * ``elexclarity 105369 GA --outputType=summary``
-* ``elexclarity 105369 GA --outputType=settings``
+* ``elexclarity 105369 GA --outputType=settings --officeID=P``
 
 Pulling formatted data from Clarity:
 
 Pass in county mapping as a JSON object.
 Sample formats can be found in the ``tests/fixtures`` folder `here <https://github.com/washingtonpost/elex-clarity/tree/develop/tests/fixtures>`_.
 
-* ``elexclarity 105369 GA --level=precinct --countyMapping='{"Worth": "13321"}``
+* ``elexclarity 105369 GA --level=precinct --countyMapping='{"Worth": "13321"}'``
 * ``elexclarity 105369 GA --level=precinct --countyMapping='<mapping json>'``
 * ``elexclarity 105369 GA --level=county --countyMapping='<mapping json>'``
 * ``elexclarity 106210 WV --level=county --countyMapping='<mapping json>'``
```

Comment:
Can't comment on the lines of code right below this, but the filepaths for fixtures are out-of-date now